src/BaseGame.ts

```diff
@@ -22,15 +22,31 @@ export default abstract class BaseGame implements IBaseGame {
      * majd kerekítsd lefelé, ez lesz az index.
      * Majd térj vissza a this.levels tömbnek ezzel az indexével.
      */
-    abstract getRandomLevel(): Level;
+     abstract getRandomLevel(levels: any[]): Level {
+        let feladat = Math.random();
+        this.levels= new Level;
+        let rand = Math.floor(feladat*this.levels.length);
+         return rand;
```

Comment:
nem kell paraméter
let helyett const
27-es sor nem kell
return this.levels[rand]